src/python/pants/option/scope.py

```diff
@@ -51,8 +54,13 @@ def deprecated_scope_removal_version(self) -> Optional[str]:
             self._subsystem_cls_attr("deprecated_options_scope_removal_version"),
         )
 
+    @property
+    def scope_aliases(self) -> Tuple[str, ...]:
+        """BuiltinGoal subsystems may define aliases."""
+        return cast(Tuple[str, ...], self._subsystem_cls_attr("aliases", ()))
+
     def _subsystem_cls_attr(self, name: str, default=None):
-        return getattr(self.subsystem_cls, name) if self.subsystem_cls else default
+        return getattr(self.subsystem_cls, name, default) if self.subsystem_cls else default
```

Comment:
Not all subsystem classes have the `aliases` attribute, and since we always return a default any way, it felt safe to provide that default already to `getattr`.